content/docs/user-guide/data-pipelines/defining-pipelines.md

```diff
@@ -0,0 +1,267 @@
+# Defining Pipelines
+
+DVC helps define programmatic workflows so that anyone can reliably
```

Comment:
These two paragraphs are two abstract.

`In a DVC project you can create a `dvc.yaml` file that has a list of commands to run to process data and get model ` - this level is better (need to rewrite it)

Reply:
OK. I rewrote the intro but I'm not sure... One goal of the guide is to give context and explain the approach, so it's OK for it to be somewhat conceptual IMO.